src/classes/AFFL_Affiliations_TDTM.cls

```diff
@@ -35,6 +35,11 @@
 * @description  Trigger handler for Affiliations related operations on Contacts, Accounts, or Affiliations.
 */
 public class AFFL_Affiliations_TDTM extends TDTM_Runnable {
+    @TestVisible
+    private static String DEFAULT_FORMER_STATUS = system.label.npe5.DefaultFormerStatus;
```

Comment:
Can you actually switch this to 'Former' and 'Current' vs using the labels. If a user is working in another language the translated value would be pulled in here otherwise.

Reply:
@jjbennett Haha that would definitely help fix the translation problem instead of just shuffling the variables around, just made the update. Good catch!